main.go

```diff
@@ -0,0 +1,75 @@
+package main
+
+import (
+	"fmt"
+	"log"
+	"net/http"
+	"os"
+
+	"github.com/gocarina/gocsv"
+	"github.com/gorilla/mux"
+)
+
+type Pokemon struct {
```

Comment:
It is always encouraged to add comments to both exported and unexported elements (such as variables, constants,  functions, methods, structs definitions, interfaces, etc.).

Please refer to https://go.dev/doc/effective_go#commentary, they explain a little about `doc comments`.

They start with the same name as the element you are describing.

In Go, the documentation of your program relies on the comments you add to describe your code.

```suggestion
// Pokemon represents a pokemon object.
type Pokemon struct {
```